### ISSUE TITLE
https://mtgjson.com/changelog/ produces a 403

### ISSUE THREAD

**Issue by Greven145** [mpaulweeks/mtgify#21]
I'm trying to integrate this great tool with a project I'm working on, and couldn't get the Kaldheim cards to load. I went to check the version details on the card database and I noticed the 403.

**Comment by mpaulweeks** [comment]
Thank you for the heads up! I'll look into it this weekend

**Comment by mpaulweeks** [comment]
Two things:

1) MTGJSON link + my version updater have been fixed! It has been updated, and now correctly reads that it's on v5 of MTGJSON. Thank you for bringing this to my attention!

2) Missing cards from new sets (in this case Kaldheim) are 90% of the time due to persistent issues with my flaky auto-updater. I am happy to update manually when I remember to (or get pinged like this), but mtgify has become very low priority for me lately. I am sadly unable to invest the time necessary to debug + fix my auto-updater in the short term. Please keep that mind when using the tool.

TL;DR Kaldheim cards should be working now, closing this as fixed. Please re-open or create a new issue if you run into any additional bugs.
